src/DebugSceneList.js

```diff
@@ -120,12 +122,18 @@ class DebugSceneList extends Component {
   }
 
   renderSceneModal() {
+    let component = this.state.selectedItem.component;
+    const closePropName = this.state.selectedItem.closePropName;
+    console.log('closePropName', closePropName);
+    if (closePropName) {
+      component = React.cloneElement(component, {[closePropName]: this.onHideScene});
+    }
     return (
```

Comment:
Good idea! In practice I always set 'name' anyway, else for production builds you end up with every item in the Component List having names like 't' and 'r'.